pkgsettings/pkgsettings.py

```diff
@@ -41,8 +76,15 @@ def configure(self, obj=None, **kwargs):
             for key, new_value in kwargs.items():
                 setattr(obj, key, new_value)
 
+        if obj is self:
```

Comment:
I think this check could be improved in order to recursively detect if there are duplicates of `self` in `obj`. The check for duplicates made below wouldn't be needed anymore.